build.sbt

```diff
@@ -12,18 +11,18 @@ inThisBuild(Seq(
 
   licenses += ("Apache 2", url("https://www.apache.org/licenses/LICENSE-2.0.txt")),
 
-  homepage := Some(url("https://outwatch.github.io/")),
+	  homepage := Some(url("https://outwatch.github.io/")),
 
-  scmInfo := Some(ScmInfo(
-    url("https://github.com/OutWatch/outwatch-libs"),
-    "scm:git:git@github.com:OutWatch/outwatch-libs.git",
-    Some("scm:git:git@github.com:OutWatch/outwatch-libs.git"))
-  )
-))
+		  scmInfo := Some(ScmInfo(
+			  url("https://github.com/OutWatch/outwatch-libs"),
+				  "scm:git:git@github.com:OutWatch/outwatch-libs.git",
+				  Some("scm:git:git@github.com:OutWatch/outwatch-libs.git"))
+			  )
+		  ))
 
```

Comment:
Why did you change the indent here? I think, it should stay as before.

Reply:
Sorry, It was not my intention, I was testing vim+metals at the same time and something happened I didn't realize.